cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -303,6 +305,10 @@ message ClusterJobStatus {
      * Job status of the node currently executing.
      */
     optional NodeJobStatus currentNode = 7;
+    /**
+     * Job specific data
+     */
+    optional string data = 8;
```

Comment:
It took me 20 minutes of digging around to figure out that this is here just to pass the name of the file to the backup job.

Please change it to be `backupFileName` or something like that so that it's more clear what the purpose of it is.  We shouldn't have any untyped containers in the project as all the work to wire in serialization has been done.